src/javascript/app_2/App/Components/Form/time_picker.jsx

```diff
@@ -239,11 +239,8 @@ TimePicker.propTypes = {
     padding        : PropTypes.string,
     placeholder    : PropTypes.string,
     sessions       : MobxPropTypes.arrayOrObservableArray,
-    start_date     : PropTypes.oneOfType([
-        PropTypes.number,
-        PropTypes.string,
-    ]),
-    value: PropTypes.string,
+    start_date     : PropTypes.number,
```

Comment:
@easteregg 
AFAIK, the value of `start_date` is not always as a number, sometimes the API returns it as a string.
IMO, it's better to handle it in the store and keep it as a number instead of handling it in components.

@negarn Would you plz confirm that I'm right or not since you added that propTypes before in [this commit](https://github.com/binary-com/binary-static/commit/a8d55766792ed98d7b9f715987257a4a06878625#diff-ad9770f97c3ed3130087a62755367212R243)?

Reply:
your suggestion sounds better, I don't remember why string was allowed before.

Reply:
@morteza-binary Which API call returns `start_date`?

Reply:
@easteregg 
Actually, there isn't a direct API call for it, the response of `contracts_for` contains `forward_starting_options` that we use it to create `trade_store.start_dates_list`. 
It seems that the best place to fix the issue is in [line 15 of `start_date.js`](https://github.com/binary-com/binary-static/blob/master/src/javascript/app_2/Stores/Modules/Trading/Helpers/start_date.js#L15) just by adding a `+` before `option.date`.